Formula/fontforge.rb

```diff
@@ -101,6 +101,18 @@ def install
     end
   end
 
+  def caveats
+    <<-EOS.undent
```

Comment:
I also preferred the way before, though as long as it’s consistent it doesn’t bother me at all.

My reasoning: though the previous way needs an extra line and extra indentation (both undesirable things), this way use a `;` (absolutely a matter of personal preference, but they’re a bit uglier to me in ruby) and the terminating `EOS` is aligned with the text (the alternative would be to align it with the `end`, but that also seems wrong).

But again, consistency above preference.